xray/utils/resultwriter.go

```diff
@@ -20,9 +21,18 @@ const (
 	Table      OutputFormat = "table"
 	Json       OutputFormat = "json"
 	SimpleJson OutputFormat = "simple-json"
+	Sarif      OutputFormat = "sarif"
 )
 
-var OutputFormats = []string{string(Table), string(Json), string(SimpleJson)}
+var OutputFormats = []string{string(Table), string(Json), string(SimpleJson), string(Sarif)}
+
+// Converting severities to their relevant score in range
+var severitiesScore = map[string]string{
+	"Critical": "9", // Critical Score Range is 9.0 – 10.0
```

Comment:
The CVEs (if existed) in an XRAY issue contain a CVSS V3 score.
I suggest running over all CVEs in the XRAY issue and finding the maximum between all V3 scores.
In some cases, there may be no CVEs at all - in such cases, we can use a dummy rank like this.

Reply:
Done - In case of license, there will be a 'Warning' sign instead of 'High'/'Critical'/... 
Does that make sense?